js/services/scrobbler.js

```diff
@@ -6,13 +6,33 @@
 
       service.getSongInformation = function () {
         var deferred = $q.defer();
+        var songInformation = {};
         if (config.lastfm.user.length) {
```

Comment:
This way, the value 'false' can be expressed. If "config.lastfm.user" is a blank string or null, "if(config.lastfm.user)" becomes false. So, if you want to check whether the user has set up the key or not, wouldn't it be more clearer to use "if(config.lastfm.user)" than the 'length' property? Also, if I do not set up the value of  "config.lastfm.user", this error comes out.
![2016-07-17 9 00 15](https://cloud.githubusercontent.com/assets/4694139/16898508/78f23dc0-4c17-11e6-856d-3a7418f1fb24.png)